src/replica.rs

```diff
@@ -120,8 +120,8 @@ impl Replica {
     }
 
     /// Get the replica's PK set
-    pub fn replicas_pk_set(&self) -> Result<PublicKeySet> {
-        Ok(self.peer_replicas.clone())
+    pub fn replicas_pk_set(&self) -> Option<PublicKeySet> {
```

Comment:
Why are they Option-ized when we return them directly?

Reply:
It had a `Resul`t, so that was just to follow the api convention (queries return `Option`, cmds return `Result`).
Then of course, if an Actor _cannot_ be instantiated without replica pks (which is the case now), there is no need for an Option. But, I saw that as a non important aspect, and decided to keep the api consistent.